### PR TITLE
feat(job-executor/execStatus): inject timezone from related schedule

### PR DESCRIPTION
**Pull request by badrAZ**
See vatesfr/xo-web#2133

### CODE REVIEW

File: src/job-executor.js [hunk 1]
```diff
@@ -146,9 +147,12 @@ export default class JobExecutor {
 
     connection.set('user_id', job.userId)
 
+    const { timezone } = find(await this.xo.getAllSchedules(), ['job', job.id])
+
     const execStatus = {
       runJobId,
       start: Date.now(),
+      timezone,
       calls: {}
```
Comment (julien-f) [inline src/job-executor.js]: Sort properties.

File: src/job-executor.js [hunk 2]
```diff
@@ -146,9 +147,12 @@ export default class JobExecutor {
 
     connection.set('user_id', job.userId)
 
+    const { timezone } = find(await this.xo.getAllSchedules(), ['job', job.id])
```
Comment (julien-f) [inline src/job-executor.js]: What if there are no schedules for this job?

Comment (julien-f) [inline src/job-executor.js]: Use `{ job: job.id }` as predicate instead of the array form.

Comment (badrAZ) [inline src/job-executor.js]: It's an error, i will fix it.

**Pull request merged**